cmd/node/flags.go

```diff
@@ -382,6 +382,26 @@ func applyFlags(ctx *cli.Context, cfgs *config.Configs, log logger.Logger) {
 	applyCompatibleConfigs(isInImportMode, importDbNoSigCheckFlag, log, cfgs.GeneralConfig, cfgs.P2pConfig)
 	flagsConfig.IsInImportMode = isInImportMode
 	flagsConfig.ImportDbNoSigCheckFlag = importDbNoSigCheckFlag
+	flagsConfig.ImportDbDirectory = importDbDirectoryValue
+
+	// if FullArchive is enabled, we override the conflicting StoragePruning settings and StartInEpoch as well
```

Comment:
please remove L387-L404 as they are done in the applyCompatibleConfigs function

Reply:
yes, I think one of the merges dupplicated the code